app/scripts/migrations/106.test.ts

```diff
@@ -0,0 +1,251 @@
+import {
+  ACCOUNT_LABEL_NAME_TYPE,
+  ACCOUNT_LABEL_CHAIN_ID,
+} from '../lib/setupAccountLabelsPetnamesBridge';
+import { migrate, version } from './106';
+
+const oldVersion = 105;
+
+describe('migration #106', () => {
+  it('updates the version metadata', async () => {
+    const oldStorage = {
+      meta: { version: oldVersion },
+      data: {},
+    };
+
+    const newStorage = await migrate(oldStorage);
+
+    expect(newStorage.meta).toStrictEqual({ version });
+  });
+
+  it('does nothing if no preferences state', async () => {
+    const oldState = {
+      OtherController: {},
+    };
+
+    const transformedState = await migrate({
+      meta: { version: oldVersion },
+      data: oldState,
+    });
+
+    expect(transformedState.data).toEqual(oldState);
+  });
+
+  it('does nothing if no account id entries', async () => {
+    const oldState = {
+      OtherController: {},
+      PreferencesController: {
+        identities: {},
+      },
+    };
+
+    const transformedState = await migrate({
+      meta: { version: oldVersion },
+      data: oldState,
+    });
+
+    expect(transformedState.data).toEqual(oldState);
+  });
+
+  const ADDRESS_1 = '0xc0ffee254729296a45a3885639AC7E10F9d54979';
```

Comment:
Minor, but should we put all the constants at the top of the file for visibility? I think that's our standard elsewhere.